torch/onnx/symbolic_opset12.py

```diff
@@ -90,17 +95,20 @@ def _dropout_returns_masked_input_and_mask(
     return r, mask
 
 
-@symbolic_helper.parse_args("v", "f", "i")
+@symbolic_helper.parse_args("v", "f", "b")
+@_beartype.beartype
 def dropout(g, input, p, train):
```

Comment:
Changed many "i" to "b" because (1) they are bool types (2) when translated to onnx Booleans are casted to int w/o issues.